meta-iot2000-bsp/recipes-core/images/wic-image.inc

```diff
@@ -1,5 +1,3 @@
-IMAGE_FSTYPES_append_iot2000 = " wic"
+IMAGE_FSTYPES_append = " wic"
 
 WKS_FILE = "wic-image.${MACHINE}.wks"
-
-WKS_FILE_DEPENDS_BOOTLOADERS_x86 = "systemd-boot"
```

Comment:
The second hunk should be a applied when switching from systemd-boot to efibootguard.

Reply:
Done.